src/types/markers.js

```diff
@@ -9,17 +9,52 @@ import type { IndexIntoStackTable, IndexIntoStringTable } from './profile';
 
 // Provide different formatting options for strings.
 export type MarkerFormatType =
-  | 'string' // "Label: Some String"
-  | 'bytes' // "Label: 5mb"
+  // ----------------------------------------------------
+  // String types.
+
+  // Show the URL, and handle PII sanitization.
+  | 'url'
+  // Show the file path, and handle PII sanitization.
```

Comment:
Do we handle these fields during sanitization already? If not, it would be good to add an issue number to the comment.

Reply:
I filed #2757